gf-webapp-admin/src/model/room.ts

```diff
@@ -7,11 +7,7 @@ export interface IRoom {
   promo?: string,
   rating?: number
   services?: string,
-  slider?: string,
-  slider2?: string,
-  slider3?: string,
-  slider4?: string,
-  slider5?: string,
+  slider?: string[],
```

Comment:
Need to delete option parameters (sign ? near the all properties)

Reply:
Cant solve this problem at the moment